src/test/java/net/sf/jabref/importer/fileformat/BiblioscapeImporterTest.java

```diff
@@ -34,7 +34,7 @@ public void testGetCLIID() {
 
     @Test
     public void testImportEntriesAbortion() throws Throwable {
-        try (InputStream is = BiblioscapeImporter.class.getResourceAsStream("BiblioscapeImporterTest2.txt")) {
+        try (InputStream is = BiblioscapeImporter.class.getResourceAsStream("BiblioscapeImporterTestAbortion.txt")) {
```

Comment:
"Abortion" indicates an exception or a real worst case. (Double check: [dict.cc's translation of Programmabbruch](http://www.dict.cc/deutsch-englisch/Programmabbruch.html))

Maybe "InvalidFile" is the right suffix?